docs/installation/config.rst

```diff
@@ -184,6 +196,29 @@ Other settings
 * ``IS_HTTPS``: Used to construct absolute URLs and controls a variety of
   security settings. Defaults to the inverse of ``DEBUG``.
 
+  .. warning::
+
+     We strongly recommended setting IS_HTTPS=False in development environments
```

Comment:
I know what you mean here, but I wonder if it could be misinterpreted when municipalities have 'dev environments' :thinking: So maybe you could add 'locally' in this sentence somewhere?

Reply:
good ditinction - I've updated it, can you check if this is better?